bindings/py/src/nupic/bindings/regions/PyRegion.py

```diff
@@ -404,3 +406,31 @@ def executeMethod(self, methodName, args):
       raise Exception('Command: ' + methodName + ' must be callable')
 
     return m(*args)
+
+  @staticmethod
+  def setSparseOutput(outputs, name, value):
```

Comment:
Does this replace another function when you are using sparse links?

Reply:
This replaces direct assignment which will not update the sparse len value.
See  the difference in the `compute` methods of the [dense](https://github.com/numenta/nupic.core/pull/1399/files/4e800a64160f74f5d8c6f7900e6663fc63e780ff#diff-d6147e3ebf8b3c547ad8f24d9534487dR158) region and [sparse](https://github.com/numenta/nupic.core/pull/1399/files/4e800a64160f74f5d8c6f7900e6663fc63e780ff#diff-d6147e3ebf8b3c547ad8f24d9534487dR91) region